sponsors/migrations/0037_add_added_by_user_sponsorbenefit.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.24 on 2021-08-19 13:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sponsors', '0036_auto_20210826_1930'),
```

Comment:
Some additional database migrations were introduced in #1852, so I think once we update this migration this is good to go! Sorry for the delay in review.

Reply:
I fixed the migrations in https://github.com/python/pythondotorg/pull/1830/commits/6b218e546e50fceb1ade902bf18c0bcc95257833